### PR TITLE
Bug 1960465: [release-4.6] Bring new etcd alerts 

### PR DESCRIPTION
**Pull request by lilic**
cc @openshift/openshift-team-monitoring please take a look thanks! @hexfusion

### CODE REVIEW

File: jsonnet/jsonnetfile.json [hunk 1]
```diff
@@ -4,11 +4,11 @@
     {
       "source": {
         "git": {
-          "remote": "https://github.com/coreos/etcd",
+          "remote": "https://github.com/etcd-io/etcd",
           "subdir": "Documentation/etcd-mixin"
         }
       },
-      "version": "master"
+      "version": "release-3.4"
```
Comment (lilic) [inline jsonnet/jsonnetfile.json:11]: FYI: there is no more branch master, as mentioned on prom-operator slack channel, etcd renamed it main now. This is the version we use in 4.6 so pinned to that.

**Comment by lilic** [comment]
The issue I think seems to be that kube-prom is pinned against a specific commit...

**Comment by hexfusion** [comment]
LGTM

**Comment by simonpasquier** [comment]
/hold
we need https://github.com/openshift/thanos/pull/57 to be merged.

**Comment by lilic** [comment]
Thanks @simonpasquier! Was odd, as it was green before and it passed test. Let me know if you need anything else from my side!

**Comment by simonpasquier** [comment]
@lilic nothing to worry from your side.

**Comment by simonpasquier** [comment]
/hold cancel

The Thanos PR has merged.

**Pull request merged**